## Next-Gen Sequence Analysis Workshop (2016)


This is the schedule for the [2016 MSU NGS course](http://bioinformatics.msu.edu/ngs-summer-course-2016). All course communications will be organized around the [Slack channel](https://anguscourse.slack.com).

This workshop has a code-of-conduct, do read it!

[Download all of these materials](https://github.com/ngs-docs/angus/archive/2016.zip) or [visit the GitHub repository](https://github.com/ngs-docs/angus/tree/2016).

Meal Times: Breakfast 7:45-8:45, Lunch 12-1, Dinner 6-7 (Unless noted below). See [HERE](http://conference.kbs.msu.edu/conference-center/food-and-beverage-service/) for the menu and other info.


### Schedule
| Day | Schedule |
|:--------|:--------|
|Monday 8/8 | 1:30PM lecture: [Welcome!](_static/2016-lecture1-welcome.pptx) (Meg and Matt)|         
| |  2:30PM [Getting Started with AWS](amazon/index) (Matt) |                  
|| 4:00PM [Intro to Linux](setup) and [Online Tutorial](http://rik.smith-unna.com/command_line_bootcamp/) (Matt) |



### People

#### Instructors:

-   Meg Staton    [@HardwoodGenomic](https://twitter.com/HardwoodGenomic?lang=en).
-   Matt MacManes [@macmanes](https://twitter.com/macmanes?lang=en).
-   Ian Dworkin [@IanDworkin](https://twitter.com/IanDworkin?lang=en).
-   Shaun Jackman [@sjackman](https://twitter.com/sjackman?lang=en).
-   Rob Patro [@nomad421](https://twitter.com/nomad421?lang=en).
-   Torsten Seemann
    [@torstenseemann](https://twitter.com/torstenseemann?lang=en).
-   Sonal Singhal

#### TAs:

-   Amanda Charbonneau
-   Lisa Cohen
-   Ming Chen
-   Will Pitchers

#### She Who Drives Many Places:

-   Kate MacManes

### Genome Assembly References
--------------------------

-   [Ben Langmead
    Resources](http://www.langmead-lab.org/teaching-materials/)
-   [How do we assemble genomes
    (videos)](https://www.youtube.com/watch?list=PLQ-85lQlPqFNGdaeGpV8dPEeSm3AChb6L&v=vjB6nhOu3BY)

### Papers and References
---------------------

#### Books

-   [Bioinformatics Data
    Skills](http://shop.oreilly.com/product/0636920030157.do)
-   [Practical Computing for Biologists](http://practicalcomputing.org/)

    These books, especially the 1st, are highly recommended book for
    people looking for a systematic presentation on shell scripting,
    programming, UNIX, etc.

### RNAseq

-   [Differential gene and transcript expression analysis of RNA-seq
    experiments with TopHat and
    Cufflinks](http://www.ncbi.nlm.nih.gov/pubmed/22383036), Trapnell et
    al., Nat. Protocols.

    One paper that outlines a pipeline with the tophat, cufflinks,
    cuffdiffs and some associated R scripts.

-   [Statistical design and analysis of RNA sequencing
    data.](http://www.ncbi.nlm.nih.gov/pubmed/20439781), Auer and
    Doerge, Genetics, 2010.
-   [A comprehensive comparison of RNA-Seq-based transcriptome analysis
    from reads to differential gene expression and cross-comparison with
    microarrays: a case study in Saccharomyces
    cerevisiae.](http://www.ncbi.nlm.nih.gov/pubmed/?term=22965124)
    Nookaew et al., Nucleic Acids Res. 2012.
-   [Challenges and strategies in transcriptome assembly and
    differential gene expression quantification. A comprehensive in
    silico assessment of RNA-seq
    experiments](http://www.ncbi.nlm.nih.gov/pubmed/?term=22998089)
    Vijay et al., 2012.
-   [Computational methods for transcriptome annotation and
    quantification using
    RNA-seq](http://www.ncbi.nlm.nih.gov/pubmed/21623353), Garber et
    al., Nat. Methods, 2011.
-   [Evaluation of statistical methods for normalization and
    differential expression in mRNA-Seq
    experiments.](http://www.ncbi.nlm.nih.gov/pubmed/?term=20167110),
    Bullard et al., 2010.
-   [A comparison of methods for differential expression analysis of
    RNA-seq data](http://www.biomedcentral.com/1471-2105/14/91), Soneson
    and Delorenzi, BMC Bioinformatics, 2013.
-   [Measurement of mRNA abundance using RNA-seq data: RPKM measure is
    inconsistent among
    samples.](http://www.ncbi.nlm.nih.gov/pubmed/?term=22872506), Wagner
    et al., Theory Biosci, 2012. Also see [this blog
    post](http://blog.nextgenetics.net/?e=51) explaining the paper in
    detail.

### Computing and Data

-   [A Quick Guide to Organizing Computational Biology
    Projects](http://www.ploscompbiol.org/article/info%3Adoi%2F10.1371%2Fjournal.pcbi.1000424),
    Noble, PLoS Comp Biology, 2009.
-   [Willingness to Share Research Data Is Related to the Strength of
    the Evidence and the Quality of Reporting of Statistical
    Results](http://software-carpentry.org/blog/2012/05/the-most-important-scientific-result-published-in-the-last-year.html),
    Wicherts et al., PLoS One, 2011.
-   [Got
    replicability?](http://econjwatch.org/articles/got-replicability-the-journal-of-money-credit-and-banking-archive),
    McCullough, Economics in Practice, 2007.

Also see this great pair of blog posts on [organizing
projects](http://nicercode.github.io/blog/2013-04-05-projects/) and
[research
workflow](http://carlboettiger.info/2012/05/06/research-workflow.html).

### Links



#### Resources

-   [Biostar](http://biostars.org) A high quality question & answer Web site.

-   [SEQanswers](http://seqanswers.com/) A discussion and information site for next-generation sequencing.

-   [Software Carpentry lessons](http://software-carpentry.org/4_0/index.html) A large number of open and reusable tutorials on the shell,
    programming, version control, etc.

#### Blogs

- [Genomes Unzipped](http://www.genomesunzipped.org/)


- [CTB Blog](http://ivory.idyll.org/blog/)

- [Getting Genetics Done](http://gettinggeneticsdone.blogspot.com/)

- [Omics! Omics!](http://omicsomics.blogspot.com/)

- [The Genome Factory](http://thegenomefactory.blogspot.com/)

